corehq/apps/sms/messages.py

```diff
@@ -34,6 +34,11 @@
 MSG_MOBILE_WORKER_JAVA_INVITATION = "sms.invitation.mobile.java"
 MSG_REGISTRATION_WELCOME_CASE = "sms.registration.welcome.case"
 MSG_REGISTRATION_WELCOME_MOBILE_WORKER = "sms.registration.welcome.mobileworker"
+MSG_UPDATE = "sms.update"
+MSG_UPDATE_LOCATION_SITE_CODE_NOT_FOUND = "sms.update.location.site_code.not_found"
```

Comment:
I think the convention here was to try and match the app translation strings, so we should try and avoid underscores. It's ok to just make `sitecodenotfound` one word at the end of this string.